src/main/webapp/WEB-INF/tags/instructor/results/filterEditModal.tag

```diff
@@ -0,0 +1,120 @@
+<%@ tag description="instructorFeedbackResultsTop - Filter Panel Edit Modal" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ attribute name="filterPanel" type="teammates.ui.template.InstructorFeedbackResultsFilterPanel" required="true" %>
+
+<div id="editModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">Edit settings</h4>
```

Comment:
Can the title be `Edit View`? The word `settings` is pretty vague in this context.